packages/mural-picker/src/components/mural-picker/index.tsx

```diff
@@ -262,12 +262,20 @@ const MuralPicker = ({
     }
   }, [rooms]);
 
-  const templateSearch = (q?: string) => {
-    if (q) {
-      const qEncoded = encodeURIComponent(q);
-      return apiClientRef.current.searchCrossWorkspaceTemplates({
-        q: qEncoded,
-      });
+  const templateSearch = async (searchKeyword?: string) => {
+    let templateResults: Template[] | TemplateSummary[];
+    if (searchKeyword) {
+      // eslint-disable-next-line no-shadow
```

Comment:
Why do we need this eslint exception?

Reply:
default templates is declared in the upper scope, I was going to check if we should change one of the names or if this is ok?

Reply:
Yes, just rename it here